model/participant/Participant.ttl

```diff
@@ -102,6 +112,38 @@ ids:participantRefinement
     rdfs:comment "Conditions which needs to be satisfied that a single Participant can be seen as a member of the subject Participant. For instance, all Participants with their headquarter in Europe might be a potential consumer of GDPR-releated data."@en;
 .
 
+ids:hasBusinessIdentifier
+	a owl:ObjectProperty ;
+	rdfs:domain ids:Participant ;
+	rdfs:range ids:BusinessIdentifier ;
+	rdfs:label "has business identifier"@en;
+	rdfs:comment "Company identifiers, described via ids:BusinessIdentifier."@en ;
+	.
+	
+ids:vatID
+	a owl:DatatypeProperty ;
+	rdfs:domain ids:Participant ;
+	rdfs:range xsd:string ;
+	rdfs:label "vat ID"@en ;
+	rdfs:comment "The Value Added Tax ID (VAT ID) is used in the EU to clearly identify a company or entity which is liable for Value Added Tax."@en ;
+.
+
+ids:legalForm
+	a owl:DatatypeProperty ;
+	rdfs:domain ids:Participant ;
+	rdfs:range xsd:string ;
+	rdfs:label "legal form"@en ;
+	rdfs:comment "Legal form of the participant. It is advisable to use an existing standard such as ISO-20275."@en ;
+.
+
+ids:jurisdication
```

Comment:
IMHO, the proper spelling is `jurisdiction` - please double-check and also update label and comment if required, thanks!